### ISSUE TITLE
typescript.configs['eslint-recommended'].rules is undefined

### ISSUE THREAD

**Issue by alexey-koran** [AndreaPontrandolfo/sheriff#19]
Hello

Here [eslint-recommended](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/src/configs/eslint-recommended.ts) doesn't have obj rules so it returns undefined and because of that these rules are not added to the config

https://github.com/AndreaPontrandolfo/sheriff/blob/0282739c32bbe063eb40f909dea0c0fd9b92834e/configs/eslint.config.js#L636

As a workaround for now

```js
...typescript.configs['eslint-recommended'].overrides[0].rules,
```

**Comment by AndreaPontrandolfo** [comment]
Thank you, i'll look into that.

**Comment by AndreaPontrandolfo** [comment]
@alexey-koran  fixed in ace189f294be3d269b0a0b4bfb5a07f260e4aa7c

Actually that ruleset was unnecessary, because it's already included in other typescript-eslint configs out-of-the-box. So i just ended up removing it.
It was an oversight, sorry for the inconvenience.
